cmake/vcpkg-ports/onnx/vcpkg.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "onnx",
+  "version-semver": "1.17.0",
+  "description": "Open standard for machine learning interoperability",
+  "homepage": "https://onnx.ai",
+  "license": "Apache-2.0",
+  "supports": "!uwp",
+  "dependencies": [
+    "protobuf",
```

Comment:
Is this duplicated as the { } section below?

Reply:
In the official vcpkg it is like this.  I guess it is because the next one means ONNX needs to use protoc from protobuf and the protoc binary needs to be compiled for the host arch. And this line means the protobuf lib should be built for the target arch(might be wasm or something else). And, it's true that we need to build protobuf twice because of this setting.